attributes/default.rb

```diff
@@ -40,6 +40,8 @@
 # Example:
 # default['datadog']['agent_version'] = '7.16.0'
 default['datadog']['agent_version'] = nil # nil to install latest
+# Agent flavor, acceptable values are "agent", iot-agent"
```

Comment:
nit: on the Agent [we use](https://github.com/DataDog/datadog-agent/blob/47c81996bc67aaf38b14f3cca34376dca6877fac/pkg/config/config.go#L61-L66) `agent` and `iot_agent` for the flavors, maybe we could use those values here too for consistency? I am not sure if the other ones are user-facing though, but just in case they ever are.

Reply:
I like `iot-agent` better because it matches the name of the package. The `flavor` names in the Agent code are to identify them on our side, but shouldn't be user-facing.

Reply:
Great, thanks Albert